ergo/questions/question.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+
+
+class Question(ABC):
+    """A question from a forecasting platform"""
+
+    @abstractmethod
+    def get_text(self):
```

Comment:
Not sure `get_text` is the best name here; maybe `get_description` or `get_title`?

Reply:
I don't have a strong opinion here, but if we're talking about a "question", I feel like a normal English phrase would be "the question's text" and not the "question's title" or the "question's description". Happy to change either way though.